README.md

```diff
@@ -154,14 +154,17 @@ pub fn shave(yak: usize) -> Result<(), Box<dyn Error + 'static>> {
 }
 
 pub fn shave_all(yaks: usize) -> usize {
-    // Constructs a new span named "shaving_yaks" at the TRACE level,
-    // and a field whose key is "yaks". This is equivalent to writing:
+    // Constructs a new span named "shaving_yaks" at the TRACE level, with two
+    // fields. Note the simple syntax for "yaks". This is equivalent to writing:
     //
     // let span = span!(Level::TRACE, "shaving_yaks", yaks = yaks);
     //
     // local variables (`yaks`) can be used as field values
     // without an assignment, similar to struct initializers.
-    let span = span!(Level::TRACE, "shaving_yaks", yaks);
+    //
+    // Also note the quotation marks for the guid:x-request-id, needed because
+    // it is not a simple token.
+    let span = span!(Level::TRACE, "shaving_yaks", "guid:x-request-id" = "abcdef", yaks);
```

Comment:
I don't think it's necessary to show the syntax for literal field names in the basic usage example in the README. This example is kind of intended to demonstrate the basics of using `tracing`, and I'd prefer to show more advanced usage elsewhere.

It's probably better to show this usage in the documentation for the `span!` and `event!` macros, which you've already added,, and maybe add an example in the `examples` dir as well.

Reply:
I've added an example, its hopefully ok.